src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -165,7 +167,7 @@ class NatsConnection implements Connection {
 
     // Connect is only called after creation
     void connect(boolean reconnectOnConnect) throws InterruptedException, IOException {
-        if (options.getServers().size() == 0) {
+        if (options.getServers().isEmpty()) {

```

Comment:
Intellij incessantly annoys me about these so I fix them when I'm in a source file.